src/test/java/io/github/minecraftcursedlegacy/testmixin/MixinPlayer.java

```diff
@@ -0,0 +1,13 @@
+package io.github.minecraftcursedlegacy.testmixin;
+
+import org.spongepowered.asm.mixin.Mixin;
+
+import net.minecraft.entity.player.Player;
+
+@Mixin(Player.class)
+public class MixinPlayer {
```

Comment:
Can we get rid of this?

Reply:
it's in the src/test set

Reply:
It doesn't do anything but it saves having to add a MixinPlayer class next time I want to make a test mixin related to player actions